assets/terraform-modules/tinkerbell-sandbox/assets/deploy/tls/Dockerfile

```diff
@@ -1,7 +1,8 @@
-FROM alpine:3.11
+FROM registry.fedoraproject.org/fedora:34
```

Comment:
@surajssd It doesn't belong in this PR, would it be trouble to take it out in a separate PR. ?

Reply:
Sure, doing it.